### PR TITLE
Allow directory of payload files

### PR DESCRIPTION
**Pull request by ballard26**
Continuation of https://github.com/redpanda-data/openmessaging-benchmark/pull/115

### CODE REVIEW

File: benchmark-framework/.factorypath [hunk 1]
```diff
@@ -1,84 +1,139 @@
 <factorypath>
```
Comment (travisdowns) [inline benchmark-framework/.factorypath:1]: I don't even really know what this files does, does anyone?

Comment (ballard26) [reply benchmark-framework/.factorypath:1]: I have no idea to be honest... something about annotations in Java? I was just carrying over the commit from the previous PR. I can just drop it.

Comment (travisdowns) [reply benchmark-framework/.factorypath:1]: I would drop it, I think it was inadvertently added. Other than that, looks good.

**Review by travisdowns — CHANGES_REQUESTED** [review]
drop factorypath

**Review by travisdowns — APPROVED** [review]

**Pull request merged**